lib/semian/resource.rb

```diff
@@ -51,5 +51,9 @@ def semid
     def key
       '0x00000000'
     end
+
+    def in_use?
```

Comment:
As discussed with @sirupsen, if `SEMIAN_SEMAPHORES_DISABLED=1`, we won't instantiate the SysV semaphore and will, therefore, use the Ruby class. In this case, it is safe to return `false` since if it's in use, we will be using the C-extension which has the `in_use?` method.